src/js/menu/menu-item.js

```diff
@@ -51,15 +63,22 @@ class MenuItem extends Button {
    * @method selected
    */
   selected(selected) {
-    if (selected) {
-      this.addClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',true);
-    } else {
-      this.removeClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',false);
+    if (this.selectable) {
```

Comment:
Yes, just using a `<select>` should hopefully makes things easier, though, it'll make even less sense to have the caption settings menu button in it then, so, we'll have to put it elsewhere.
Also, `<select>` is *not* styleable at all.